modules/hw-installing-amq-interconnect-messaging-bus.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * monitoring/using-rfhe.adoc
+
+:_content-type: PROCEDURE
+[id="hw-installing-amq-interconnect-messaging-bus_{context}"]
+= Installing the AMQ messaging bus
+
+To pass Redfish hardware event notifications between publisher and subscriber on a node, you must install and configure an AMQ messaging bus to run locally on the node. You do this by installing the AMQ Interconnect Operator for use in the cluster.
+
+.Prerequisites
+
+* Install the {product-title} CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+* Install the AMQ Interconnect Operator to its own `amq-interconnect` namespace. See link:https://access.redhat.com/documentation/en-us/red_hat_amq/2021.q1/html/deploying_amq_interconnect_on_openshift/adding-operator-router-ocp[Installing the AMQ Interconnect Operator].
+
+.Verification
+
+. Verify that the AMQ Interconnect Operator is available and the required pods are running:
++
+[source,terminal]
+----
+$ oc get pods -n amq-interconnect
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                    READY   STATUS    RESTARTS   AGE
+amq-interconnect-645db76c76-k8ghs       1/1     Running   0          23h
+interconnect-operator-5cb5fc7cc-4v7qm   1/1     Running   0          23h
+----
+
+. Verify that the required `bare-metal-event-relay` hardware event producer pod is running in the `openshift-bare-metal-events` namespace:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-bare-metal-events
```

Comment:
A general comment: 
In Openshift documentation, we use `bare metal` in text and descriptions, and we use `baremetal` instead of `bare-metal` in names of components and services, for example, `openshift-baremetal-events`.

Reply:
Understood. However this is not what we've come to understand from @nishant-parekh

Reply:
@jzding as the name agreed upon by management was: bare metal event relay.. My understanding is that it will translate to the hyphenated version of this like bare-metal-event-relay.. so we would want to make sure that naming convention is reflected accordingly..

Reply:
I don't see any other components in openshift hyphenated as `bare-metal-xxx`. The `baremetal-xxx` seems to be a convention:
https://github.com/openshift/release/search?q=baremetal